Add definition for Faker.js

joeybaker opened the pull request:
Creates a new definition for [faker.js](https://github.com/Marak/faker.js)

JelteF commented on definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js:
```diff
@@ -0,0 +1,261 @@
+      firstName: (gender?: mixed) => string,
+      lastName: (gender?: mixed) => string,
+      findName: (
+        firstName: string,
```
this should be `firstName?: ?string,`, because it's optional

joeybaker replied:
Huh. The docs indicate it's required: https://marak.github.io/faker.js/faker.name.html#-static-findName__anchor, but the code shows that its not: https://github.com/Marak/faker.js/blob/3a4bb358614c1e1f5d73f4df45c13a1a7aa013d7/lib/name.js#L68-L77

Thanks! Will change.

joeybaker replied:
Done.

AndrewSouthpaw commented on definitions/npm/faker_v4.x.x/test_faker_v4.x.x.js:
```diff
@@ -0,0 +1,311 @@
+import faker from "faker";
```
Move the test file into the versioned flow directory.

AndrewSouthpaw commented on definitions/npm/faker_v4.x.x/test_faker_v4.x.x.js:
```diff
@@ -0,0 +1,311 @@
+import faker from "faker";
+
+faker.address.zipCode();
```
Use `it/describe` blocks for more winning.

joeybaker commented:
@AndrewSouthpaw thanks for the review! I moved the file and added a few `describe` blocks. I'm not sure there's much benefit to breaking down the tests more.
